src/table/cells/handle-key-press.js

```diff
@@ -59,31 +59,31 @@ export const preventDefaultBehavior = (evt) => {
   evt.preventDefault();
 };
 
-export const moveFocus = (evt, rootElement, cellCoord, focusedCellCoord, selState) => {
+export const moveFocus = (evt, rootElement, cellCoord, selState, setFocusedCellCoordsState) => {
   preventDefaultBehavior(evt);
   removeCurrentFocus(evt);
   const rowAndColumnCount = getRowAndColumnCount(rootElement);
   const nextCellCoord = arrowKeysNavigation(evt, rowAndColumnCount, cellCoord, selState);
   updateFocus(rowAndColumnCount.rowElements, nextCellCoord);
-  focusedCellCoord.current = nextCellCoord;
+  setFocusedCellCoordsState(nextCellCoord);
 };
 
 export const headHandleKeyPress = (
   evt,
   rootElement,
   cellCoord,
-  focusedCellCoord,
   changeSortOrder,
   layout,
   isDim,
-  isAnalysisMode
+  isAnalysisMode,
+  setFocusedCellCoordsState
 ) => {
   switch (evt.key) {
     case 'ArrowUp':
```

Comment:
Just realized we don't need to do anything for arrowUp, since you are. already at the top. You can remove it :)

Reply:
Good Catch ✅
it's because we already are at the head of table we don't need to move upper that that 👌